src/vxingest/grib2_to_cb/grib_builder_parent.py

```diff
@@ -665,70 +665,62 @@ def build_document(self, queue_element):
                 self.ds_translate_item_variables_map = {
                     "2 metre temperature": ds_hgt_2_metre_temperature.variables[
                         list(ds_hgt_2_metre_temperature.data_vars.keys())[0]
-                        if len(list(ds_hgt_2_metre_temperature.data_vars.keys())) > 0
-                        else None
-                    ],
+                    ]
+                    if len(list(ds_hgt_2_metre_temperature.data_vars.keys())) > 0
+                    else None,
```

Comment:
Ah, this is a nice catch. We want the map value equal to none (`ds_translate_item_variables_map["2 metre temperature"] == None`) when there aren't any keys in `ds_hgt_2_metre_temperature.data_vars`, correct?

Reply:
yes